flask_restplus/api.py

```diff
@@ -207,8 +216,6 @@ def documentation(self, func):
 
     def swagger_view(self):
```

Comment:
You're right, the factory function is not necessray anymore.
We can extract SwaggerView somewhere (make it more reuseable) and call:
```python
self._register_view(
    app_or_blueprint,
    SwaggerView,
    '/swagger.json',
    endpoint=str('specs'),
    resource_class_args=(self, )
)
```